griptape/processors/amazon_comprehend_processor.py

```diff
@@ -0,0 +1,33 @@
+from typing import Callable, Optional, List, Dict, Any, Union
+from attr import define, field, Factory
+from .base_processors import BasePromptStackProcessor
+import boto3
+from botocore.client import BaseClient
+
+
+@define
+class AmazonComprehendPiiProcessor(BasePromptStackProcessor):
+    comprehend_client: BaseClient = field(default=Factory(lambda: boto3.client("comprehend")), kw_only=True)
```

Comment:
Please use `session` to instantiate the client. See `AmazonBedrockPromptDriver` for reference.